app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ContinueOrDeletePage.scala

```diff
@@ -28,9 +27,9 @@ object ContinueOrDeletePage {
 
   implicit val format: OFormat[ContinueOrDeletePage] = derived.oformat()
 
-  def unapply(str: String): Option[ContinueOrDeletePage] = str match {
-    case IsFalseish() => Some(Skip)
-    case IsTrueish()  => Some(Show)
-    case _            => None
+  def unapply(str: Boolean): Option[ContinueOrDeletePage] = if (str) {
+    Some(Show)
+  } else {
+    Some(Skip)
```

Comment:
This never returns `None` so there is no need for `Option[ContinueOrDeletePage]` as a return type

Reply:
There is a pattern matching [here](https://github.com/hmrc/gform/blob/main/app/uk/gov/hmrc/gform/sharedmodel/formtemplate/DraftRetrievalMethod.scala#L31) I think we should return it as Option

Reply:
> There is a pattern matching [here](https://github.com/hmrc/gform/blob/main/app/uk/gov/hmrc/gform/sharedmodel/formtemplate/DraftRetrievalMethod.scala#L31) I think we should return it as Option

In provided link we can get rid of matching on `showContinueOrDeletePage: Boolean` and use id directly.

So we should provide this 

```
def fromBoolean(str: Boolean): ContinueOrDeletePage = if (str) Show else Skip
```
instead of this `unapply` method